p2panda-rs/src/test_utils/constants.rs

```diff
@@ -7,10 +7,9 @@
 pub const DEFAULT_HASH: &str =
     "0020b177ec1bf26dfb3b7010d473e6d44713b29b765b99c6e60ecbfae742de496543";
 
-/// The default schema hash string, used in all operation fixtures when no custom schema hash is
-/// defined.
-pub const DEFAULT_SCHEMA_HASH: &str =
-    "0020c65567ae37efea293e34a9c7d13f8f2bf23dbdc3b5c7b9ab46293111c48fc78b";
+/// Unpublished test schema id, used in all operation fixtures that don't define their own schema.
+pub const TEST_SCHEMA_ID: &str =
+    "venue_0020c65567ae37efea293e34a9c7d13f8f2bf23dbdc3b5c7b9ab46293111c48fc78b";
```

Comment:
I changed this constant name to not make it sound like there is a _default_ schema in p2panda. Would like to do the same with the other constants here.